From b31fedb06538b968227895b32cf3423a9ca29f99 Mon Sep 17 00:00:00 2001
From: Judith Mendez <jm@ti.com>
Date: Wed, 21 Feb 2024 18:33:14 -0600
Subject: [PATCH 12/27] arm64: dts: ti: k3-am62a7-sk: Enable GPIOs in GPIO1
 bank

Enable remaining GPIOs in GPIO1 bank for k3-am62a7-sk-rpi-hdr-ehrpwm
overlay.

While we are at it, remove MCASP0 node (which disabled MCASP0) since
it is not enabled by default in the board DTS file.

Signed-off-by: Judith Mendez <jm@ti.com>
---
 .../boot/dts/ti/k3-am62a7-sk-rpi-hdr-ehrpwm.dtso   | 14 ++++++++++----
 1 file changed, 10 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/ti/k3-am62a7-sk-rpi-hdr-ehrpwm.dtso b/arch/arm64/boot/dts/ti/k3-am62a7-sk-rpi-hdr-ehrpwm.dtso
index 1d0d7bca08e5..d1294eeddb6b 100644
--- a/arch/arm64/boot/dts/ti/k3-am62a7-sk-rpi-hdr-ehrpwm.dtso
+++ b/arch/arm64/boot/dts/ti/k3-am62a7-sk-rpi-hdr-ehrpwm.dtso
@@ -27,7 +27,17 @@
 
 	rpi_header_gpio1_pins_default: rpi-header-gpio1-pins-default {
 		pinctrl-single,pins = <
+			AM62AX_IOPAD(0x0194, PIN_INPUT, 7) /* (C19) MCASP0_AXR3.GPIO1_7 */
+			AM62AX_IOPAD(0x0198, PIN_INPUT, 7) /* (B19) MCASP0_AXR2.GPIO1_8 */
+			AM62AX_IOPAD(0x01a4, PIN_INPUT, 7) /* (A19) MCASP0_ACLKX.GPIO1_11 */
+			AM62AX_IOPAD(0x01b4, PIN_INPUT, 7) /* (D16) SPI0_CS0.GPIO1_15 */
+			AM62AX_IOPAD(0x01b8, PIN_INPUT, 7) /* (C16) SPI0_CS1.GPIO1_16 */
+			AM62AX_IOPAD(0x01bc, PIN_INPUT, 7) /* (A17) SPI0_CLK.GPIO1_17 */
+			AM62AX_IOPAD(0x01c0, PIN_INPUT, 7) /* (B15) SPI0_D0.GPIO1_18 */
+			AM62AX_IOPAD(0x01c4, PIN_INPUT, 7) /* (E15) SPI0_D1.GPIO1_19 */
 			AM62AX_IOPAD(0x01d0, PIN_INPUT, 7) /* (F14) UART0_CTSn.GPIO1_22 */
+			AM62AX_IOPAD(0x01d8, PIN_INPUT, 7) /* (B17) MCAN0_TX.GPIO1_24 */
+			AM62AX_IOPAD(0x01dc, PIN_INPUT, 7) /* (C18) MCAN0_RX.GPIO1_25 */
 		>;
 	};
 
@@ -74,10 +84,6 @@
 	};
 };
 
-&mcasp0 {
-	status = "disabled";
-};
-
 &epwm0 {
 	status = "okay";
 	pinctrl-names = "default";
-- 
2.39.2

